client/src/client_proxy.rs

```diff
@@ -6,15 +6,14 @@ use admission_control_proto::proto::admission_control::SubmitTransactionRequest;
 use config::{config::PersistableConfig, trusted_peers::ConsensusPeersConfig};
 use crypto::{ed25519::*, test_utils::KeyPair};
 use failure::prelude::*;
-use futures::{future::Future, stream::Stream};
-use hyper;
```

Comment:
Since we're no longer using hyper directly mind removing the hyper crate from the Cargo.toml file?

Reply:
yep, I removed both hyper and tokio